### DISCUSSION TITLE
How to make responsive PnP search results

### DISCUSSION THREAD

**Discussion by supreet-bajaj** [microsoft-search/pnp-modern-search#2589] [Ideas]
How we can make responsive our PnP search results page? 
It's not looking good in mobile phones. Please see attached screenshots
<img width="215" alt="image" src="https://user-images.githubusercontent.com/25117777/203871307-fd5f805d-d333-41d0-bbad-233b3e23660b.png">
<img width="152" alt="image" src="https://user-images.githubusercontent.com/25117777/203871655-8bb01b28-1d5d-4d3c-90bf-d14859747f35.png">

Below is laptop layout version for your reference.
<img width="725" alt="image" src="https://user-images.githubusercontent.com/25117777/203871588-09edf54b-eed3-4ce2-ad72-fece73f16460.png">

Can someone please confirm if we can achieve responsive search results page.
Thanks,
Supreet Bajaj

**Comment by wobba** [comment]
Do a PR and include CSS which works better on responsive :) We haven't invested much into this, which means we need assistance on the OOB templates. Or you create a custom template with the CSS you need.
